### PR TITLE
update various URLs/copyrights to the new pycqa/modernize project

### PR DESCRIPTION
**Pull request by graingert**
this also pushes more into the plain `modernize` name

### CODE REVIEW

File: docs/fixers.rst [hunk 1]
```diff
@@ -223,70 +223,70 @@ version of ``six`` is installed.
...
 
    If :func:`zip <python2:zip>` is called, ``from six.moves import zip`` is added to the module.
    Wrapping the use in a call to ``list()`` is done when necessary.
 
 
-``2to3`` fixers
+``fissix`` fixers
 +++++++++++++++
 
 Some `fixers from fissix <https://docs.python.org/3/library/2to3.html#fixers>`_
```
Comment (graingert) [inline docs/fixers.rst:235]: @jreese I'd like to reference the fissix docs here https://github.com/jreese/fissix/issues/23

File: docs/fixers.rst [hunk 2]
```diff
@@ -316,7 +316,7 @@ to specify the ``-f default`` or ``--fix=default`` option, e.g.::
 
    This is intended for use in programs where ``/`` is conventionally only used
    for integer division, or where it is intended to do a manual pass after running
-   python-modernize to look for cases that should not have been changed to ``//``.
+   ``python -m odernize`` to look for cases that should not have been changed to ``//``.
```
Comment (jreese) [inline docs/fixers.rst:319]: missing an "m"

**Review by jreese — APPROVED** [review]

File: docs/fixers.rst (hunk 2 above)
Comment (graingert) [reply docs/fixers.rst]: thanks!

File: docs/fixers.rst [hunk 3]
```diff
@@ -232,7 +232,7 @@ version of ``six`` is installed.
 ``fissix`` fixers
 +++++++++++++++
 
-Some `fixers from fissix <https://fissix.readthedocs.io/en/latest/fixers.html>_`
+Some `fixers from fissix <fissix:fixers>`
```
Comment (graingert) [inline docs/fixers.rst:235]: this link isn't working and neither are the links from the fixers

Comment (altendky) [reply docs/fixers.rst:235]:
```suggestion
Some :ref:`fixers from fissix <fissix:fixers>`
```
I think...

File: README.rst [hunk 4]
```diff
@@ -10,11 +10,11 @@ This library is a very thin wrapper around `fissix
 to make Python 2 code more modern with the intention of eventually
 porting it over to Python 3.
 
-The ``python-modernize`` command works like `2to3
-<https://docs.python.org/3/library/2to3.html>`_. Here's how you'd rewrite a
+The ``python -m modernize`` command works like `fissix
+<https://github.com/jreese/fissix>`_. Here's how you'd rewrite a
```
Comment (jreese) [inline README.rst:14]: Maybe word this different, since fissix doesn't (currently) have a command line entrypoint like 2to3.

**Review by jreese — APPROVED** [review]

File: README.rst (hunk 4 above)
Comment (graingert) [reply README.rst:14]:
```suggestion
The ``python -m modernize`` command works like
``python -m fissix``, see `fissix <https://github.com/jreese/fissix>`_.
Here's how you'd rewrite a
```

@jreese see also https://github.com/jreese/fissix/pull/24/files

**Pull request merged**